rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -189,10 +189,16 @@ object RSpace {
       sc: Serialize[C],
       sp: Serialize[P],
       sa: Serialize[A],
-      sk: Serialize[K]): RSpace[C, P, A, R, K] = {
-    val mainStore = LMDBStore.create[C, P, A, K](context, branch)
-    create(mainStore, branch)
-  }
+      sk: Serialize[K]): RSpace[C, P, A, R, K] =
+    context match {
```

Comment:
Just so I understand this correctly (sorry if asking dumb question. We are given here a `context` as parameter and we create one again?